app/commands/submission/analysis/process.rb

```diff
@@ -52,6 +55,8 @@ def submission
 
   memoize
   def data
+    return nil if ops_errored?
```

Comment:
```suggestion
    return {} if ops_errored?
```

(Same as the exception below?)

Reply:
I saw that the database column was nullable, and as we probably don't want to process errored records anyway, so I thought nil. Still want `{}`?

Reply:
I think it should be the same as the exception clause. That's my only hard rule. I think I prefer `{}` over `nil` just to avoid the sort of errors that we were getting here, but I'm less precious about that bit as long as they're both the same.